examples/react-template/screens/Select.tsx

```
@@ -108,6 +108,31 @@ export const SelectView = (): JSX.Element => {
+              status='success'
+              iconName='tri-alert'
+              name='option'
+              label='label'
+              id='select-native-id'
+              selected={option4}
+              onChange={(e) => {
+                e?.selectValue && setOption4(e.selectValue)
+              }}
+            >
+              {[...Array(20).keys()].map((_, i) => {
+                return <SelectOption id={`option_${i}`} value={`option_${i}`} label={`option_${i}`} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)